website/docs/community/join.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Join the community
+id: join
+---
+
+<section className="community-home">
+
+Want to learn how organizations around the world are tackling the biggest challenges in data while making new friends from the best analytics teams? Join the dbt Community — data practitioners’ favorite place to learn new skills, keep on top of industry trends, and forge connections.
+
+<div className="grid--3-col">
+
+<Card
+    title="Join us on Slack"
+    body="Follow the pulse of the dbt Community! Chat with other practitioners in your city, country or worldwide about data work, tech stacks, or simply share a killer meme."
+link="https://www.getdbt.com/community/join-the-community/"
+    icon="slack"
+/>
+
+<Card
+    title="Community Forum"
+    body="Have a question about how to do something in dbt? Hop into the Community Forum and work with others to create long lived community knowledge."
+    link="forum"
+    icon="discussion"
+/>
```

Comment:
It looks like the forum page is now at `/community/forum`, the link for this card should be updated

Reply:
oooh good catch lemme search for that and events which both movied